docs/CloudPermissions.md

```diff
@@ -0,0 +1,56 @@
+# Restricting cloud privileges for deploying Scalar DL
+
+This guide explains how to restrict privileges for deploying Scalar DL on Azure.
+
+In general from security perspective, it is better to assign restricted privileges to users not to make them able to do operations that they are not supposed to do.
+
+## Azure
+
+The following JSON is a custom role that allow users to manage resources for Scalar DL. This is sufficient but not necessary.
+
+In Azure Portal, you can create the role in Subscriptions section. Choose your subscription and select Access control (IAM) from the menu, then click `+Add` and select `Add custom role`. 
```

Comment:
What about specifying it with bullet points?

Reply:
>What about specifying it with bullet points?

Thank you for the suggestion.
I used [Scalar-terraform cloud permission document ](https://github.com/scalar-labs/scalar-terraform/blob/master/docs/CloudPrivileges.md) as a reference to create this document. Should I change this?